packages/react-pdf-toolkit/.storybook/preview.tsx

```diff
@@ -0,0 +1,15 @@
+import type { Preview } from '@storybook/react';
+
+const preview: Preview = {
+  parameters: {
+    actions: { argTypesRegex: '^on[A-Z].*' },
+    controls: {
+      matchers: {
+        color: /(background|color)$/i,
+        date: /Date$/,
+      },
+    },
+  },
+};
```

Comment:
Objects of similar nature, configs, etc work best with `satisfies`.

```suggestion
} satisfies Preview;
```